packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3087,17 +3087,21 @@ Polls a push verification transaction for completion. The transaction result is
 {
   "expiresAt": "2015-04-01T15:57:32.000Z",
   "factorResult": "WAITING",
+  "profile":{
+     "credentialId":"jane.doe@example.com",
+     ...
+  },
   "_links": {
     "poll": {
-      "href": "https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/opfh52xcuft3J4uZc0g3/transactions/mst1eiHghhPxf0yhp0g",
+      "href": "https://${yourOktaDomain}/api/v1/users/00u15s1KDETTQMQYABRL/factors/opfh52xcuft3J4uZc0g3/transactions/v2mst.GldKV5VxTrifyeZmWSQguA",
```

Comment:
Is this the new transactionId prefix, or something like that?

Reply:
i think so, this is what I observed in my testing, is there a way to confirm it?

Reply:
Ah yes, looks like this is right:
https://github.com/okta/okta-core/blob/aa15de77430cd7c93fd30f4ad47b24dc891e52e1/server/core/src/main/java/com/saasure/core/services/auth/StashAuthStateTokenStore.java#L34

My bad, nothing came up when I searched it on IntelliJ earlier. I think I just had a bad setting/filter on or something.

Reply:
thank you for confirming :)